site/docs/tasks/custom-taxonomy.md

```diff
@@ -20,15 +20,15 @@ This task describes how to deploy Fybrik with a custom `taxonomy.json` file that
 
 ## Taxonomy Compile CLI tool 
 
-A CLI tool for compiling a base taxonomy and zero or more taxonomy layers is provided in our repo.
+A CLI tool for compiling a base taxonomy and zero or more taxonomy layers is provided in the [taxonomy-cli repository](https://github.com/fybrik/taxonomy-cli), along with a [Docker image](https://github.com/fybrik/taxonomy-cli/pkgs/container/taxonomy-cli) to directly run the tool.
```

Comment:
we don't have to provide a link to th econtaine/taxonomy-cli, you can just mention the docker image.

Reply:
done